config/ftbquests/quests/chapters/basic_power.snbt

```diff
@@ -0,0 +1,1164 @@
+					id: "503B730A6C1C60E0"
+					type: "xp"
+					xp: 100
+				}
+				{
+					id: "61435729546E13B6"
+					type: "random"
+					exclude_from_claim_all: true
+					table_id: 487623848494439020L
+				}
+			]
+		}
+		{
+			icon: "powah:dielectric_casing"
+			x: -2.0d
+			y: 3.0d
+			shape: "hexagon"
+			description: [
+				"You can't power your base off of Coal forever! If you've got a decent amount of resources available, it's time to upgrade your power setup!"
+				""
+				"These options usually require a little setting up, but produce enough power to carry you far into your playthrough."
+			]
+			size: 1.5d
+			id: "5E41363F9AE243F3"
+			tasks: [{
+				id: "5A8F474927CC1E80"
+				type: "checkmark"
+				title: "Mid Game Power Options"
+			}]
+		}
+		{
+			title: "&eMekanism's&r &dGas-Burning Generator&r"
+			x: -2.0d
+			y: 4.5d
+			shape: "rsquare"
+			subtitle: "The Power of the Melon"
+			description: [
+				"If you've ever heard of someone talking about \"Melon Power\", this is it. Mekanism's &9Gas-Burning Generator&r can produce a good amount power by pumping in &9Ethylene&r made from Melon Slices."
+				""
+				"To produce &dEthylene&r, start by crushing organic materials in a &eCrusher&r to create &6Bio Fuel&r. Melon Slices are typically used for this! This is then pumped into a &dPressurized Reaction Chamber&r (PRC for short)."
+				""
+				"The PRC needs Water, Bio Fuel, and some Hydrogen to create Ethylene. You can get the Hydrogen from separating water in an &9Electrolytic Separator&r. "
+				""
+				"Once you've started producing the Ethylene, pump it into the Gas-Burning Generator to start generating power!"
+			]
+			dependencies: ["5E41363F9AE243F3"]
+			min_width: 300
+			id: "0AEC181F5E21A299"
+			tasks: [
+				{
+					id: "2ADE2B487637FA3F"
+					type: "item"
+					item: "mekanismgenerators:gas_burning_generator"
+				}
+				{
+					id: "5C1F3A52B007EDB9"
+					type: "item"
+					item: "mekanism:pressurized_reaction_chamber"
+				}
+				{
+					id: "41DDBC04B42A604E"
+					type: "item"
+					item: "mekanism:crusher"
+				}
+				{
+					id: "59CC349D61EE0729"
+					type: "item"
+					item: "mekanism:electrolytic_separator"
+				}
+			]
+			rewards: [
+				{
+					id: "7B9C300DF6442A1D"
+					type: "item"
+					item: "minecraft:melon_slice"
+					count: 8
+				}
+				{
+					id: "742C6774E1F40C96"
+					type: "xp"
+					xp: 100
+				}
+				{
+					id: "542C81754D93237B"
+					type: "random"
+					exclude_from_claim_all: true
+					table_id: 4196188979167302596L
+				}
+			]
+		}
+		{
+			title: "Powah"
+			icon: "powah:book"
+			x: -1.0d
+			y: 4.0d
+			shape: "rsquare"
+			description: [
+				"Believe it or not, the Powah mod has some great options for getting... powah."
+				""
+				"The &9Reactors&r are great options for reliable, consistent power, as long as you have the coal and Uraninite to power it. You will need &936 Reactor Blocks&r in total to create a reactor. Just right-click and it will build itself in the world!"
```

Comment:
```suggestion
				"The &9Reactors&r are great options for reliable, consistent power, as long as you have the coal, redstone, water, ice and Uraninite to power it. You will need &936 Reactor Blocks&r in total to create a reactor. Just right-click and it will build itself in the world!"
```
The reactor also accepts Redstone, Water, and Ice for increased production and efficiency.

Reply:
There will be a Powah Chapter later down the road to explain the use of the other mats, as I believe those are not required.